classes/class-twenty-twenty-one-custom-colors.php

```diff
@@ -105,7 +105,7 @@ public function editor_custom_color_variables() {
 			array(),
 			(string) filemtime( get_theme_file_path( 'assets/css/custom-color-overrides.css' ) )
 		);
-		if ( 'D1E4DD' !== get_theme_mod( 'background_color', 'D1E4DD' ) ) {
+		if ( 'd1e4dd' !== strtolower( get_theme_mod( 'background_color', 'D1E4DD' ) ) ) {
```

Comment:
Any reason the first hex code is lowercase now?

Reply:
> Any reason the first hex code is lowercase now?

Since users can manually enter any color they want, they may enter lowercase, uppercase or mixed. Using lowercase here makes sense because we're passing the theme-mod through `strtolower` which will convert it to lowercase. This way the case used by users won't matter. It's more robust this way :+1: